README.md

```diff
@@ -391,6 +391,30 @@ A function to start your application.  When invoked, `this` will be your instanc
 - `modulePackage`: the contents of the local module's package.json (if found)
 - `configFiles`: an object of filepaths for each found config file (filepath values will be null if not found)
 
+
+### prepareConfig(env)
+
+Requiring `env.configPath` and modules specified by CLI options or config files are executed here. This method gives your application a chance to change the `env` params before these requiring and to load `env.configFiles`.
+
+```js
+Hacker.prepareConfig = function(env) {
```

Comment:
@sttk how is this going to be used in gulp-cli?  Also, which issue is it solving?

Reply:
I'll use it for the issuee 181 in gulp-cli as follows:

```js
cli.prepareConfig = function(env) {
  var cfgLoadOrder = ['home', 'cwd'];
  var cfg = loadConfigFiles(env.configFiles['.gulp'], cfgLoadOrder);
  opts = mergeConfigToCliFlags(opts, cfg);
  env = mergeConfigToEnvFlags(env, cfg);
  env.configProps = cfg;

  Liftoff.prototype.prepareConfig.call(this, env);
};
```

Reply:
I'm going to think about this API a bit more.  We might want to split the prepare and launch phases and bump another major version.  I'll follow up this week.

Reply:
I got it.